ex5.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+
+    <script>
+
+        var userInput1 = prompt("Peroana 1");
+        var userSalary1 = prompt("Whats your salary?");
+        var userInput2 = prompt("Peroana 2");
+        var userSalary2 = prompt("Whats your salary?");
+        var userInput3 = prompt("Peroana 3");
+        var userSalary3 = prompt("Whats your salary?");
+
+        function averageSalary(userSalary1, userSalary2, userSalary3) {
+
+            var sumSalary = (parseInt(userSalary1) + parseInt(userSalary2) + parseInt(userSalary3));
+            var average = parseFloat(sumSalary) * 12 / 3;
+
+            console.log(sumSalary);
+            alert(average);
+
+            if (userSalary1 > userSalary2 && userSalary1 > userSalary3) {
```

Comment:
The aim was to compare the salaries with the average, and find the closest one. This can be achieved by calculating the difference between each salary and the average, in absolute value(`Math.abs`), and then compare the differences.